library/patient.inc

```diff
@@ -1261,7 +1263,10 @@ function newInsuranceData(
     }
 
   // If a bad date was passed, err on the side of caution.
-    $effective_date = fixDate($effective_date, date('Y-m-d'));
+    $effective_date = DateToYYYYMMDD($effective_date);
+    if (empty($effective_date)) {
+        $effective_date = '0000-00-00';
+    }
 
```

Comment:
TODO
Test this out

Reply:
TODO
Seem odd that this is being done within functions. Seems better to do these conversion on the script (before the date parameter is brought into the function; I will look into this).

Reply:
TODO
revert above changes and have this instead done in origin scripts

Reply:
DONE